### ISSUE TITLE
[NEWS] Obsługa przesyłek - zmiana w identyfikatorach przewoźników.

### ISSUE THREAD

**Issue by PawelTaberski** [allegro/allegro-api#1957]
**Od 12 sierpnia** dla zamówień z metodą dostawy:

- Allegro Pocztex Kurier 48
- Pocztex Kurier 24
- Pocztex Kurier 48.

Zamiast identyfikatora POCZTEX, w poniższych procesach przekaż identyfikator Poczta Polska:

- w zasobie [POST /order/checkout-forms/{checkoutForm.id}/shipments](https://developer.allegro.pl/documentation/#operation/getOrderShipmentsUsingGET) **przesyłaj w polu carrierId: wartość “POCZTA_POLSKA”,**
- w metodzie [doAddPackageInfoToPostBuyForm](https://allegro.pl/webapi/documentation.php/show/id,744) **przesyłaj w polu operatorId wartość 6 - Poczta Polska.**

**Comment by kormichu** [comment]
Czy identyfikator POCZTEX będzie nadal do czegoś wykorzystywany po Waszej stronie ? Jeśli nie to może lepiej byłoby zrobić identyfikator POCZTEX jako alias do POCZTA_POLSKA. Obyłoby się wtedy bez problemów

**Comment by PawelTaberski** [comment]
Identyfikator ten będzie usuwany i nie będzie już można z niego korzystać po 12 sierpnia.

**Comment by SebastianOzdoba** [comment]
@PawelTaberski to bym bardziej proponuję by zachować kompatybilność, przy wysłaniu 15 zamieniać go u Was na 6 - Poczta-Polska. Wtedy macie nad tym kontrolę a my nie musimy hardkodować zmian, bo nie ma metody słownikowej gdzie ShipmentId -> CarrierId

**Comment by PawelTaberski** [comment]
Rozumiem, oczywiście sugestię przekażę, niemniej raczej i tak jak już wcześniej pisałem identyfikator ten będzie usuwany i nie będzie już można z niego korzystać po 12 sierpnia.

**Comment by SebastianOzdoba** [comment]
@PawelTaberski nie bedzie go można używać u Was, ale w innych integracjach jak najbardziej Pocztex będzie możliwy :)

**Comment by SebastianOzdoba** [comment]
To jeszcze jedno pytanie @PawelTaberski , w komunikacie są 3 sposoby dostawy, a co ze sposobem dostawy: Pocztex Kurier48 Pobranie, shipmentid = 22, jak rozumiem też ma być PocztaPolska?

**Comment by PawelTaberski** [comment]
Tak, zmiana dotyczy zarówno przesyłek płatnych z góry jak i pobraniowych.
